### DISCUSSION TITLE
trying to pull in wind data from GFS using fast_Herbie_xarray

### DISCUSSION THREAD

**Discussion by v7nguyen** [blaylockbk/Herbie#82] [General]
I am attempting to pull wind components from GFS but fast_Herbie_xarray is failing to run since it returns multiple hypercubes. Does anyone know the proper search string to use in order to execute this functionality correctly? The search string I am using currently is "(?:U|V)GRD:10 m".
![image](https://user-images.githubusercontent.com/53784737/174811935-52659170-d5e5-4f7a-93bf-67c0c68732ca.png)

**Comment by blaylockbk** [comment]
I think for GFS, that particular search string matches both ***10 meters*** and ***10 mb***. Can you try it with the search string `"(?:U|V)GRD:10 m above ground"`

**Reply by v7nguyen** [reply]
I got it to work with `"(?:U|V)GRD:10 m above ground"`! The only issue now is that the query returns a dataset but the latitude/longitude columns are missing...
![image](https://user-images.githubusercontent.com/53784737/175098244-a8324a7d-c2d0-4d78-affd-f9c0c7d4209c.png)
![image](https://user-images.githubusercontent.com/53784737/175098388-5afc5316-9c9c-4514-b17d-909ae884bcd7.png)

**Reply by blaylockbk** [reply]
Ah, you are right. I'll update my previous comment. `(?:U|V)GRD:10 m above ground` is the right searchString. 

The lat/lon are stored as coordinates of the xarray DataSet.
![image](https://user-images.githubusercontent.com/6249613/175165082-158f1bc3-a508-4a5d-9b6e-5bf5f1a52a73.png)

I see you are turning the DataSet into a Pandas DataFrame. The latitude and longitude values should be in the multi-index. Try `df = ds.to_dataframe().reset_index()`

![image](https://user-images.githubusercontent.com/6249613/175165183-e17581b0-e901-4900-9a39-ebea91e6c696.png)

![image](https://user-images.githubusercontent.com/6249613/175165201-53a15c2b-889d-44ea-8ea7-51a21a07a2b6.png)
